.pre-commit-config.yaml

```diff
@@ -5,6 +5,11 @@ repos:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 21.6b0
+    hooks:
+    - id: black
+      language_version: python3
```

Comment:
Are you sure this is necessary?

Reply:
you mean the python3 config or the black pre-commit config as whole?

Reply:
`pre-commit` support is not strictly necessary, so we can remove.

Reply:
I don't think I have an opinion on this. Any option would do.